README.md

```diff
@@ -18,27 +18,51 @@ npm install --save-dev @cypress/browserify-preprocessor
 
 ## Usage
 
+### Prior to Cypress 10.0
+
 In your project's [plugins file](https://on.cypress.io/plugins-guide):
 
 ```javascript
-const browserify = require('@cypress/browserify-preprocessor')
+const browserify = require("@cypress/browserify-preprocessor");
 
 module.exports = (on) => {
-  on('file:preprocessor', browserify())
-}
+  on("file:preprocessor", browserify());
+};
+```
+
+### Cypress 10.0 and Beyond
+
+In your project's [cypress config](https://on.cypress.io/plugins-api):
```

Comment:
Is this the correct link to /plugins-api?

Reply:
@jordanpowell88 , everything else looks good to me. So once this link is fixed, or you let me know that it is correct, I will approve it.